RESPONSIBILITIES.md

```diff
@@ -51,19 +52,25 @@ Review pull requests regularly, comment, suggest, reject, merge and close. Accep
 
 ### Triage Open Issues
 
-Manage labels, review issues regularly, and triage by labelling them. 
+Manage labels, review issues regularly, and triage by labelling them.
 
 All repositories in this organization have a standard set of labels, including `bug`, `documentation`, `duplicate`, `enhancement`, `good first issue`, `help wanted`, `blocker`, `invalid`, `question`, `wontfix`, and `untriaged`, along with release labels, such as `v1.0.0`, `v1.1.0`, `v2.0.0`, `patch`, and `backport`.
 
 Use labels to target an issue or a PR for a given release, add `help wanted` to good issues for new community members, and `blocker` for issues that scare you or need immediate attention. Request for more information from a submitter if an issue is not clear. Create new labels as needed by the project.
 
 ### Be Responsive
 
-Respond to enhancement requests, and forum posts. Allocate time to reviewing and commenting on issues and conversations as they come in. 
+Respond to enhancement requests, and forum posts. Allocate time to reviewing and commenting on issues and conversations as they come in.
 
 ### Maintain Overall Health of the Repo
 
-Keep the `main` branch at production quality at all times. Backport features as needed. Cut release branches and tags to enable future patches. 
+Keep the `main` branch at production quality at all times. Backport features as needed. Cut release branches and tags to enable future patches.
+
+#### Keep Dependencies up to Date
+
+Maintaining up-to-date dependencies on third party projects reduces the risk of security vulnerabilities. The Open Source Security Foundation (OpenSSF) [recommends](https://github.com/ossf/scorecard/blob/c26bea648d0ef252de841ba59a55e8d67fe30271/docs/checks.md#dependency-update-tool) either [dependabot](https://docs.github.com/en/code-security/dependabot) or [renovatebot](https://docs.renovatebot.com/).
+ - Dependabot is integrated with GitHub and can be enabled simply by adding a [`dependabot.yml`](https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuring-dependabot-version-updates) file to the repo. This will create individual pull requests for each dependency update that can be tracked via tags.
+ - Renovatebot requires installation of the [`forking-renovate`](https://github.com/apps/forking-renovate) app which will generate a pull request to set itself up. The dependency update pull requests have more information on version age, adoption rate, and percent of successful tests of those who have upgraded. In addition, a centralized dashboard issue provides visibility into all available version updates even if the PR has not yet been generated.
```

Comment:
Is the app required on a repo? When I follow that link I don't get opensearch-project in my options of orgs to select. How do I enable it for my repo inside of opensearch-project? Example of a repo where this was enabled?

Reply:
It requires an internal ticket. I'm not sure how much information I should be putting on github about internal processes.   Also since it's now enabled for the org in order to apply to our repo I think that means it's now enabled for the organization and anyone can add the config file.  I'll do some testing on my other projects and find out if this is the case...

Reply:
Played around on my other project. It does require per-repo permissions and the org admin does need to grant those permissions per-repo to generate the config PR.